region/region-readme.md

```diff
@@ -32,19 +32,26 @@ The `utility` property is an array of utility functions available to Samus at a
 * _super_
 * _powerbomb_
 * _energy:_ Note that this excludes reserve tanks
-* _reserve_: Note that this excludes regular energy. If a node can refill reserves as well as energy, it will have both `energy` and `reserve`
+* _reserve:_ Note that this excludes regular energy. If a node can refill reserves as well as energy, it will have both `energy` and `reserve`
 * _map_
-* _farming_: Represents the presence monster spawners. This may be removed later and replaced by actual monster spawners.
-#### unlock
-The `unlock` property lists [logical requirements](../logicalRequirements.md) that must be fulfilled to properly interact with a node. Note that unlike traversing links, `unlocking` a node is an action that needs to be done only once. The interaction locked behind `unlock` requirements can take several forms such as:
+* _farming:_ Represents the presence monster spawners. This may be removed later and replaced by actual monster spawners.
+#### locks
+The `locks` property is an array that contains different ways a node can be locked, and the corresponding way it can be unlocked. Each object in the `locks` property has two properties of its own:
+* _lock:_ The `lock` property lists [logical requirements](../logicalRequirements.md) that must be fulfilled in order for the node to be locked. Ig this is missing, the node is considered initially locked at game start.
+* _unlock:_ The `unlock` property lists [logical requirements](../logicalRequirements.md) that must be fulfilled in order to undo this specific lock.
+
+__Additional considerations:__ None of the locks must be active for Samus to be able to properly interact with a node. Note that unlike traversing links, `unlocking` a lock is an action that needs to be done only once. Interacting with a node, which requires no locks to be active, can take several forms such as:
 * Using a door node to go to another room
 * Picking up the item at an item node
 * Completing an event node's event
+* Using any `utility` that is present at a note
```

Comment:
node

Reply:
Wonder how many times I'm mess one of those two words up